recipes/tixi3/all/conandata.yml

```
@@ -0,0 +1,8 @@
+sources:
+  "3.3.0":
+    sha256: "988d79ccd53c815d382cff0c244c0bb8e393986377dfb45385792766adf6f6a9"
+    url: "https://github.com/DLR-SC/tixi/archive/refs/tags/v3.3.0.tar.gz"
+patches:
+  "3.3.0":
+    - patch_file: "patches/link_curl.patch"
+    - patch_file: "patches/disable_tixi_examples.patch"
```

```suggestion
    - patch_file: "patches/disable_tixi_examples.patch"
      description: "Keep build examples as optional"
      patch_type: "conan"
```
https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches

---

The conan commit hook errors with your proposed changes:
[HOOK - conan-center.py] pre_export(): ERROR: [CONANDATA.YML FORMAT (KB-H030)] Additional entries ['description'] not allowed in 'patches':'3.3.0' of conandata.yml (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H030)

What should I do?

---

sorry, it's patch_description